src/BulkAppointments/RecurringAppointments.js

```diff
@@ -12,7 +14,9 @@ const calculateEndDate = ({ startTime, repeats, numberOfTimes }) => {
   }
 
   const unitOfTime = repeatsToMomentMap[repeats]
-  const times = Number(repeats === 'FORTNIGHTLY' ? Number(numberOfTimes) * 2 : Number(numberOfTimes))
+
+  const InclusiveAdjustment = numberOfTimes - 1
```

Comment:
lowercase variable should start with lowercase?

Reply:
Ah good spot

Reply:
Ah good spot